components/calculator/rows/row.tsx

```diff
@@ -1,18 +1,30 @@
 import type { Ingredient } from "../../../src/utils/api";
+import styles from "./row.module.css";
 
 type RowProps = {
   options: Ingredient[];
+  selectedIng: string;
+  onSelectChange: (selection: string) => void;
 };
 
-function Row({ options }: RowProps) {
+function Row({ options, selectedIng, onSelectChange }: RowProps) {
   return (
     <div>
       <div>
```

Comment:
Not related to this pull request, but a single `div` should be enough :)

Reply:
Will do thank you! I always end up having way too many <div> 😅